tests/e2e/monitoring/test_monitoring_on_negative_scenarios.py

```diff
@@ -0,0 +1,426 @@
+from ocs_ci.ocs.monitoring import (
+    check_pvcdata_collected_on_prometheus,
+    check_ceph_health_status_metrics_on_prometheus,
+    prometheus_health_check
+)
+from ocs_ci.ocs.node import (
+    wait_for_nodes_status, get_node_objs,
+    get_typed_nodes, drain_nodes, schedule_nodes
+)
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import CommandFailed, ResourceWrongStatusException
+
+log = logging.getLogger(__name__)
+
+
+@retry(AssertionError, tries=30, delay=3, backoff=1)
+def wait_to_update_mgrpod_info_prometheus_pod():
+    """
+    Validates the ceph health metrics is updated on prometheus pod
+
+    """
+
+    log.info(
+        f"Verifying ceph health status metrics is updated after rebooting the node"
+    )
+    ocp_obj = ocp.OCP(kind=constants.POD, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    mgr_pod = (
+        ocp_obj.get(selector=constants.MGR_APP_LABEL).get('items')[0].get('metadata').get('name')
+    )
+    assert check_ceph_health_status_metrics_on_prometheus(mgr_pod=mgr_pod), (
+        f"Ceph health status metrics are not updated after the rebooting node where the mgr running"
+    )
+    log.info("Ceph health status metrics is updated")
+
+
+@retry((CommandFailed, TimeoutError, AssertionError, ResourceWrongStatusException), tries=10, delay=3, backoff=1)
+def wait_for_nodes_status_and_prometheus_health_check(pods):
+    """
+    Waits for the all the nodes to be in running state
+    and also check prometheus health
+
+    """
+
+    # Validate all nodes are in READY state
+    wait_for_nodes_status(timeout=900)
+
+    # Check for the created pvc metrics after rebooting the master nodes
+    for pod_obj in pods:
+        assert check_pvcdata_collected_on_prometheus(pod_obj.pvc.name), (
+            f"On prometheus pod for created pvc {pod_obj.pvc.name} related data is not collected"
+        )
+
+    assert prometheus_health_check(), "Prometheus health is degraded"
+
+
+@tier4
+@ignore_leftovers
+@workloads
+class TestMonitoringBackedByOCS(E2ETest):
+    """
+    Test cases to validate monitoring backed by OCS
+    """
+    num_of_pvcs = 5
+    pvc_size = 5
+
+    @pytest.fixture(autouse=True)
+    def init_sanity(self):
+        """
+        Initialize Sanity instance
+
+        """
+        self.sanity_helpers = Sanity()
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, nodes):
+        """
+        Restart nodes that are in status NotReady, for situations in
+        which the test failed before restarting the node after detach volume,
+        which leaves nodes in NotReady
+
+        """
+
+        def finalizer():
+            not_ready_nodes = [
+                n for n in get_node_objs() if n
+                .ocp.get_resource_status(n.name) == constants.NODE_NOT_READY
+            ]
+            log.warning(
+                f"Nodes in NotReady status found: {[n.name for n in not_ready_nodes]}"
+            )
+            if not_ready_nodes:
```

Comment:
let's not do this as I see that the health of nodes are checked after each time it is rebooted.

Reply:
removed this validation, so resolving conversation